_episodes/07-branches.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Branches
+teaching: 20
+exercises: 0
+questions:
+- "What are branches?"
+- "How can I work in parallel using branches?"
+objectives:
+- "Explain why branches can be useful."
```

Comment:
Could this objective be something like "Understand why branches are useful for working on separate tasks in the same repository."?